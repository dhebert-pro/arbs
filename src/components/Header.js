import React from "react";
import { Link } from "routes";

const Header = () => <div><Link route='/'><a>Home</a></Link></div>;

export default Header;