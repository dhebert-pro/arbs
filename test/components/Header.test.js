/*global expect test describe:true */
/*eslint no-undef: "error"*/
import React from "react";
import Header from "components/Header";
import renderer from "react-test-renderer";

describe("Header", () => {
  test("Header should be rendered", done => {
    const component = renderer.create(<Header />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    done();
  });
});
