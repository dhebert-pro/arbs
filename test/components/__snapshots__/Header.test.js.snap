// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header should be rendered 1`] = `
<div>
  <a
    href="/"
    onClick={[Function]}
  >
    Home
  </a>
</div>
`;
